session-1-Local-Variable-Type-Inference/05_JShell/README.md

```diff
@@ -0,0 +1,33 @@
+**Local Variable Type Inference for all Types example**
+
+This example covers the use of `Type Inference` in JShell to declare variables using them in Expressions, Statements, and Blocks as Loops.
+
+You can run this example on your local JShell or online 
+
+To execute Local JShell run next command
+
+     jshell
+     
+Verify you are using JShell for Java 10 version
+
+    |  Welcome to JShell -- Version 10
+    |  For an introduction type: /help intro
+
+
+To execute JShell online go to next URL
```

Comment:
Add link to a whole array of java shells as shared at https://github.com/neomatrix369/jdk9-jigsaw/tree/master/session-3-jshell (see whole page for links)

Reply:
Done